plugins/org.jboss.tools.openshift.client/build.properties

```diff
@@ -2,6 +2,15 @@ source.. = src/
 src.includes = *
 src.excludes = src/
 output.. = target/classes
-bin.includes = .,\
-               META-INF/,\
-               lib/*
+bin.includes = META-INF/,\
+               .,\
+               plugin.xml,\
+               plugin.properties,\
+               about.html,\
+               about.ini,\
+               about.mappings,\
+               about.properties,\
+               jboss_about.png,\
+               .options,\
+               lib/*,\
+               log4j.properties
```

Comment:
I thought the file has changed following our switch to Log4J2 ?

Reply:
ouch, too quick again, thanks for spotting.